### PR TITLE
New attempt of workaround SGE + Java incompatibility for eager @ EVA

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

File: conf/pipeline/eager/eva.config [hunk 1]
```diff
@@ -3,7 +3,7 @@
 params {
   // Specific nf-core/configs params
   config_profile_contact = 'James Fellows Yates (@jfy133)'
-  config_profile_description = 'nf-core/eager EVA profile provided by nf-core/configs'
+  config_profile_description = 'nf-core/eager EVA profile provided by nf-core/configs DEV'
```
Comment (TCLamnidis) [inline conf/pipeline/eager/eva.config:6]: Should this be DEV?

**Review by TCLamnidis — APPROVED** [review]
LGTM, except for the DEV in the description.
Are all these codes really needed in all the tools?

File: conf/pipeline/eager/eva.config (hunk 1 above)
Comment (jfy133) [reply conf/pipeline/eager/eva.config:6]: Not any more!

**Comment by jfy133** [comment]
Yes... don't want ot put the codes in non-Java tools as those actually work correctly... unfotunately we have a lot of java tools...

File: conf/pipeline/eager/eva.config (hunk 1 above)
Comment (jfy133) [inline conf/pipeline/eager/eva.config:6]:
```suggestion
  config_profile_description = 'nf-core/eager EVA profile provided by nf-core/configs'
```

**Comment by TCLamnidis** [comment]
What I meant was do all the Java tools need all these exit codes, or did you apply them all across the board for consistency?

**Comment by TCLamnidis** [comment]
Also, if you have a list of what each code you added means to append here, I would not be against it :eyes:

**Comment by jfy133** [comment]
> What I meant was do all the Java tools need all these exit codes, or did you apply them all across the board for consistency?

Consistency, but all Java ones need the exit code 1, as all their errors results in that. I kept the other things in tehe as I couldn't remeber if they are inherited or overwritten from the base exit codes in the pipeline.

> Also, if you have a list of what each code you added means to append here, I would not be against it eyes

Varies per tool slightly, they just _generally_ mean the same thing...ish...

**Pull request merged**